cp1404practicals/prac_09/cleanup_files.py

```diff
@@ -0,0 +1,48 @@
+"""
+CP1404/CP5632 Practical
+Demos of various os module examples
+"""
+import os
+import re
+
+
+def main():
+    """Demo os module functions."""
+    print("Starting directory is: {}".format(os.getcwd()))
+
+    # Change to desired directory
+    os.chdir('Lyrics/Christmas')
+
+    # Print a list of all files in current directory
+    print("Files in {}:\n{}\n".format(os.getcwd(), os.listdir('.')))
```

Comment:
Possibly separate so the '\n' is not needed like shown below (mainly just so it's a little more readable)

`print("Files in {}:".format(os.getcwd))`
`print(os.listdir(.))`
`print()`